kuma/wiki/models.py

```diff
@@ -1806,7 +1808,7 @@ def make_current(self):
     def __unicode__(self):
         return u'[%s] %s #%s: %s' % (self.document.locale,
                                      self.document.title,
-                                     self.id, self.content[:50])
```

Comment:
I find the fact we show revision content in the string version of the revision silly.